src/client/e2e/credit.spec.ts

```diff
@@ -92,4 +91,44 @@ test.describe("Credit", () => {
       expect(tradeId).toEqual(blotterId)
     })
   })
+
+  test.describe("Sell side", () => {
+    test.only("Sell side ticket", async ({ context }) => {
+      await newRfqPage.getByPlaceholder(/Enter a CUSIP/).click()
+      await newRfqPage
+        .locator("[data-testid='search-result-item']")
+        .nth(5)
+        .click()
+
+      const quantity = newRfqPage.locator("[data-testid='quantity']")
+      await quantity.type("2")
+      await quantity.blur()
+
+      await newRfqPage
+        .locator("span")
+        .getByText(/Adaptive Bank/)
+        .click()
+
+      const pagePromise = context.waitForEvent("page")
```

Comment:
minor/question .. why is `pagePromise` set up here, and awaited below ..

Reply:
This is how the docs example looks, I did try initialising it after and it didn't work, I have no idea why it has to be this way